lib/helpers/codemirror.ts

```diff
@@ -22,6 +22,10 @@ const toggleCodeBlock: Command = ({ state, dispatch }) => {
   return toggleAround(state, dispatch, codeBlockMark, codeBlockMark)
 }
 
+const toggleSpoiler: Command = ({ state, dispatch }) => {
+  return toggleAround(state, dispatch, '||', '||')
```

Comment:
:exclamation: This syntax is not supported by our markdown parser! :exclamation:

This will not work properly. [The docs shows how to properly do spoilers.](https://docs.modrinth.com/markdown#details-spoilers)